auth-api/src/auth_api/models/product_subscription.py

```diff
@@ -16,10 +16,11 @@
 The ProductSubscription object connects Org models to one or more ProductSubscription models.
 """
 
-from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy import Column, ForeignKey, Integer, and_
 from sqlalchemy.orm import relationship
 
 from .base_model import VersionedModel
+from ..utils.roles import VALID_SUBSCRIPTION_STATUSES
```

Comment:
We may need to think about a pattern for importing modules.. from a readability point of view its better to start from the root module except for files from same package..